pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -298,6 +298,9 @@ func (m *cgroupCommon) toResources(resourceConfig *ResourceConfig) *libcontainer
 	if resourceConfig.PidsLimit != nil {
 		resources.PidsLimit = *resourceConfig.PidsLimit
 	}
+	if !resourceConfig.CPUSet.IsEmpty() {
+		resources.CpusetCpus = resourceConfig.CPUSet.String()
+	}
```

Comment:
non-blocking question for better understanding: what would be the drawback, if any, in always setting the `CpusetCpus`?
this PR covers the `none` policy flow
the exclusively-allocated CPUs should be wanting to set this value anyway (we do in the hooks IIRC?)
the `static` policy without exclusively allocated CPUs should be not too hard.

Reply:
mostly as a very targeted change because I don't immediately know the consequences of setting it in the other cases and I'd rather get the bug fix through

Reply:
that's fair and reasonnable, let's keep it this way.